experimental/conventional-model/terraform/modules/ops/triggers.tf

```diff
@@ -0,0 +1,92 @@
+resource "time_sleep" "wait_for_cloud_build_service" {
+  create_duration = "20s"
+  depends_on = [
+    google_project_service.emblem_ops_services
+  ]
+}
+
+resource "google_cloudbuild_trigger" "api_unit_tests_build_trigger" {
+  project        = var.project_id
+  count          = var.deploy_triggers ? 1 : 0
+  name           = "api-unit-tests"
+  filename       = "ops/unit-tests.cloudbuild.yaml"
+  included_files = ["content-api/**"]
+  substitutions = {
+    _DIR             = "content-api"
+    _SERVICE_ACCOUNT = "restricted-test-identity@emblem-ops.iam.gserviceaccount.com"
```

Comment:
Is this not managed in Terraform? I don't see it elsewhere in this PR.

Reply:
Totally copied as is from the original [cloud_build_triggers.tf](https://github.com/GoogleCloudPlatform/emblem/blob/f0553b15786bc00efb400e1abe5923171ce6586a/terraform/ops/triggers/cloud_build_triggers.tf#L15) 

will change to a dynamic project id